gulpfile.js

```diff
@@ -436,6 +436,9 @@ function createTestSource(testsName, bot) {
     if (bot) {
       args.push("--strictVerify");
     }
+    if (process.argv.includes("--noChrome")) {
+      args.push("--noChrome");
+    }
```

Comment:
As I hoped, when asking for this feature, it didn't require a lot of code; thanks for doing this!

Just one small thing: Can you please add this same block to the `makeRef` function below as well, such that *all* the various test commands are covered?

Reply:
Done in the new commit series.